meteor/client/ui/DndListItemWrapper.tsx

```diff
@@ -0,0 +1,87 @@
+import { DragDropItemTypes } from './DragDropItemTypes'
+import React, { useRef } from 'react'
+import { DragSourceMonitor, DropTargetMonitor, useDrag, useDrop, XYCoord } from 'react-dnd'
+import { Identifier } from 'dnd-core'
+import { faGripLines } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+interface IDndItemWrapperProps {
+	index: number
```

Comment:
It seems weird to give an index to a single item. This indicates that the item is a part of a list, but this component doesn't know anything about that (and it shouldn't know anything about it)

Reply:
I think this is a leftover and that it isn't used. I will remove and test, good catch!

Reply:
I understand what you mean, but for the internal hover method to work for each specific item the index is needed. As you mentioned in an earlier comment, it wasn't completely clear that the item always belongs to a list. I will change that. I think it makes sense this way when looking at how the tables that would make use of the generic drag and drop should follow the same structure.

Reply:
Quick fix I think would be just to rename `IDndItemWrapperProps` to `DndListItemWrapperProps`. Then we know it's part of a list and the index makes sense

Reply:
It has already been changed to `IDndListItemWrapperProps` but as long as noone objects to me removing the `i`-prefix, i will change all the interface names in a later commit!